src/seedu/addressbook/data/person/Unit.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.person;
+
+public class Unit {
+     //data members
+	 private String unit;
+	 //constructor
+	 public Unit (String str){
+		 unit = str;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.